editor/plugins/tiles/tile_map_layer_editor.cpp

```diff
@@ -763,9 +763,13 @@ bool TileMapLayerEditorTilesPlugin::forward_canvas_gui_input(const Ref<InputEven
 				}
 
 			} else {
-				// Released
-				_stop_dragging();
```

Comment:
The above `_stop_dragging()` call relies on the state of `drag_erasing` for selecting patterns when right clicking.